mailing_list/models.py

```diff
@@ -168,6 +168,17 @@ def get_or_create_or_delete_mailing_lists_for_canvas_course_id(self, canvas_cour
         return result
 
 
+class CourseSettings(models.Model):
+    canvas_course_id = models.IntegerField()
+    always_mail_staff = models.NullBooleanField(null=True, default=True)
+    modified_by = models.CharField(null=True, max_length=32)
+    date_created = models.DateTimeField(null=True, default=timezone.now)
+    date_modified = models.DateTimeField(null=True, default=timezone.now)
```

Comment:
I think we want `auto_now=True` instead of `default`, right? Otherwise this will only be automatically updated on create (not on updates), I think.